tests/ui/traits/next-solver/member-constraints-in-root-universe.rs

```diff
@@ -1,16 +1,16 @@
 //@ compile-flags: -Znext-solver
+//@ check-pass
 
 trait Trait {
     type Ty;
 }
 
-impl Trait for for<'a> fn(&'a u8, &'a u8) {
+impl Trait for for<'a, 'b> fn(&'a u8, &'b u8) {
```

Comment:
There's a chance that bc this binder is equal then we don't actually check anything. Need to do a HR-eq here in order to bump the universe.

Reply:
maybe do `let _bump: for<'a> fn(&'a (), &'a ()) = (|&(), &()| ()) as for<'a, 'b> fn(&'a (), &'b ());` instead then inside of `test`, removing `Trait` entirely

Reply:
Actually this is moot because we register obligations too early to ever bump the universe after you land your changes to registering opaques early lol

Reply:
'm just gonna remove the test